test/validation/org/apache/cassandra/bridges/ArchiveClusterLogs.java

```diff
@@ -147,4 +147,11 @@ public static boolean countErrors(String errorLogs)
         else
             return false;
     }
+
+    public static String[] metaDataFilesFormatter(String files)
+    {
+        files = files.substring(1, files.length()-1);
```

Comment:
Removing first '[' and last ']' characters from the list getting from ccm getsstables